src/Elastic.Transport/Elastic.Transport.csproj

```diff
@@ -12,9 +12,11 @@
     <IsPackable>true</IsPackable>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <TargetFrameworks>netstandard2.0;netstandard2.1;net462;net8.0</TargetFrameworks>
-    <IsAotCompatible>true</IsAotCompatible>
     <JsonSerializerIsReflectionEnabledByDefault>false</JsonSerializerIsReflectionEnabledByDefault>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0'">
```

Comment:
@Mpdreamz I learned that it's better to use e.g. `$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net8.0'))` in these cases, because otherwise, the condition does not equate to `true` for "sub-frameworks" like `net8.0-ios`, etc.